spec/spec.md

```diff
@@ -139,13 +139,17 @@ sequenceDiagram
 - VCs MUST adhere to the [[ref: VC Data Model]] and be encoded as JSON and signed as JWT as defined in 6.3.1 of [[ref: VC Data Model]]. VCs encoded as JSON-LD and signed using Linked Data Proofs are NOT supported.
 - For key management and authentication, Self-Issued OpenID Connect Provider v2, an extension to OpenID Connect, MUST be used as defined in [[ref: SIOPv2]].
 - For transportation of VCs, OpenID for Verifiable Presentations MUST be used as defined in [[ref: OpenID4VP]].
-- As the query language, Presentation Exchange MUST be used and conform to the syntax defined in [[ref: OpenID4VP]], a profile of [[ref: Presentation Exchange]].
+- As the query language, Presentation Exchange v1.0.0 MUST be used and conform to the syntax defined in [[ref: OpenID4VP]], a profile of [[ref: Presentation Exchange]].
 - Decentralized Identifiers (DIDs), as defined in [[ref: DID Core]], MUST be used as identifiers of the entities. Implementations MUST support did:web and did:ion as a mandatory DID method as defined in [[ref: did-web]] and [[ref: did-ion]].
 - To bind an owner of a DID to a controller of a certain origin, a Well Known DID Configuration MUST be used as defined in [[ref: Well Known DID]].
 - For Revocation of VCs, Status List 2021 as defined in [[ref: Status List 2021]] MUST be used in combination with Identity Hubs as defined in [[ref: Identity Hub]] (Decentralized Web Node v0.0.1 predraft).
 
 It is important to note that Cross-device SIOP is susceptible to a session phishing attack, where an attacker relays the request from a good Verifier/RP to a victim and is able to sign in as a victim. Implementers MUST implement mitigations most suitable to the use-case. For more details and concrete mitigations, see section 15 Security Considerations in [[ref: SIOPv2]].
 
+Note: This profile uses First Implementer's Drafts of Self-Issued OpenID Provider v2 and OpenID for Verifiable Prensetations. This document does not necessarily rely on the latest editor's drafts of these specifications but may choose to do so anytime.
```

Comment:
is it `First Implementer's Drafts` or `first Implementer Draft` or just `Implementer's Drafts` with a link ?

Reply:
`This profile uses First Implementer's Drafts of Self-Issued OpenID Provider v2 and OpenID for Verifiable Prensetations specifications.`